WalletWasabi.Backend/Controllers/ChaumianCoinJoinController.cs

```diff
@@ -117,7 +117,7 @@ public async Task<IActionResult> PostInputsAsync([FromBody, Required] InputsRequ
 
 			using (await InputsLock.LockAsync())
 			{
-				CoordinatorRound round = Coordinator.TryGetRound(request.RoundId);
+				CoordinatorRound? round = Coordinator.TryGetRound(request.RoundId);
```

Comment:
I prefer to have the classical `TryWhatever(key, out var theThing)` pattern or at least the `if (TheObject.TryWhatever(key) is not null theThing`

Reply:
Yes, that's a good point. Changed.